__tests__/common/services/FormatService/FormatService.test.js

```diff
@@ -55,17 +55,17 @@ describe('FormatService: headerDate', () => {
   it('Formats correctly', () => {
     const formatter = new FormatService();
 
-    expect(formatter.headerDate(new Date('1/1/2020'))).toBe('01 January');
-    expect(formatter.headerDate(new Date('2/20/2020'))).toBe('20 February');
+    expect(formatter.headerDate(new Date('1/1/2020').getTime() / 1000)).toBe('01 January');
```

Comment:
I was a bit confused by this comment, but I did make the change! 
42016b9

Reply:
I was trying to say, when I see `2/20/2020` I'm confused - because I think of 'normal' as 'D/M/YYYY' but the test says that this is 20th February, which means the date string here is US format 'M/D/YYYY'. No biggie, I'm usually confused.. 🤷‍♂️

Reply:
OH of course.. ! ha